net2-manifests.yaml

```
@@ -15,7 +15,7 @@
     replace:
       path: openshift/99_openshift-cluster-api_net2_worker-machineset-0.yaml
       regexp: '^\s{6}metadata:.*'
-      replace: '      metadata:\n        labels:\n          node-role.kubernetes.io/net2: ""'
+      replace: '      metadata:\n        labels:\n          node-role.kubernetes.io/net2: ""\n          network: {{ net2ExternalNetwork | lower }}'
```

As mentioned, I wonder if this should handle the case when the Network name has spaces? It is possible for networks to have spaces but I don't think any of our Ext nets do currently.
```suggestion
      replace: '      metadata:\n        labels:\n          node-role.kubernetes.io/net2: ""\n          network: {{ net2ExternalNetwork | replace(' ','-') | lower }}'
```